From 51efcd436c30e7085b36264771edce03316013d1 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Thu, 1 Aug 2019 19:44:00 +0300
Subject: [PATCH] arm64: dts: ls1028a: Add ethernet property for l2switch CPU
 port

This enables the CPU traffic for the l2 switch (aka the
CPU frame injection/ extraction feature).

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -812,6 +812,7 @@
 					};
 					port@5 {
 						reg = <5>;
+						ethernet = <&enetc_port3>;
 						fixed-link {
 							speed = <1000>;
 							full-duplex;
@@ -819,7 +820,7 @@
 					};
 				};
 			};
-			ethernet@0,6 {
+			enetc_port3: ethernet@0,6 {
 				compatible = "fsl,enetc";
 				reg = <0x000600 0 0 0 0>;
 				fixed-link {
